intro_to_rl.ipynb

```diff
@@ -0,0 +1,3249 @@
+{
```

Comment:
<p>Spelling/Grammar:</p><p> </p><p>In this tutorial, we will be learning about Reinforcement Learning, a type of Machine Learning where an agent learns to <s>chose</s> choose actions in an environment that <s>lead</s> leads to maximal reward in the long run.</p><p> </p><p>Along the way you will be <s>introduces</s> introduced to some of the most important concepts and terminology in RL</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Resolved. Thanks.</p>